packages/api/src/routes/profileRoutes.js

```diff
@@ -1,8 +1,40 @@
 const profileRoutes = require('express').Router();
-const { User } = require('../db');
+const { User, Batch } = require('../db');
 const ERR_MSGS = require('../../constants/ERR_MSGS');
 const profileValidation = require('../services/profileValidation');
 const { isAuthenticated } = require('../helper/auth/isAuthenticated');
+const { extractUser } = require('../helper/user');
+
+profileRoutes.get('/user/:id', isAuthenticated, extractUser, async (req, res) => {
+  const { id } = req.params;
+  const { user } = req;
+  if (user) {
```

Comment:
Try to check negativity first. It'll reduce lots of indentation.
Also, it increases code readability

Reply:
Yup. changed it in next Commit.